style.css

```diff
@@ -703,6 +822,61 @@ media queries Below
     font-weight: 500;
     border-radius: 8px;
   }
+
+  /*
+  Desktop PopUP start----------------
+  */
+  .portfolio-pupUp-background {
+    padding-left: 10%;
+    padding-right: 10%;
+    box-sizing: border-box;
+  }
+
+  .porfolio_pop-up_main-container {
+    padding-top: 5%;
+    padding-left: 2%;
+    padding-right: 2%;
+  }
```

Comment:
Great design team🤝

- [ ] As stated in the project requirements, kindly ensure your design conforms to the Figma design provided. You may ensure some features of your design are not overflowing from the intended layout.
- [ ] In your design some text is overflowing, you can fix this both in your mobile and desktop popups.
- [ ] Kindly ensure the image for your desktop popup is not distorted. Ensure it appears like how it appears in the design given.
- Your design
![popup-desktop](https://github.com/Diegogagan2587/Diego-Vidal/assets/63917725/9f9a944a-ef29-4acf-b610-48bd1cd73cab)
- Given design
![Given-desktop](https://github.com/Diegogagan2587/Diego-Vidal/assets/63917725/db3675ab-ea47-422a-94be-d5b53e629024)
- [ ] Kindly also ensure your mobile popup has buttons visible just like the way they appear in the design given.
- Your design
![popu-mobile](https://github.com/Diegogagan2587/Diego-Vidal/assets/63917725/8842e8f7-f766-449a-8619-883866833aee)
- Given design
![Given mobile](https://github.com/Diegogagan2587/Diego-Vidal/assets/63917725/5e1e9cbc-b4cc-449e-b14c-245df9f6a9f6)
- [ ] You may also ensure your design is responsive both on mobile and desktop views.

Reply:
Buddy I was working on another branch yesterday that was created from this one, I finished the other feature and now this pull request is closed, Can I create another pull request? @1GORDON